src/Equations/RunawayFluid.cpp

```diff
@@ -229,7 +233,10 @@ void RunawayFluid::GridRebuilt(){
     nuD->GridRebuilt();    
 }
 
+<<<<<<< HEAD
+=======
 
+>>>>>>> master
```

Comment:
These are remains from a merge conflict and shouldn't compile?

Reply:
probably forgot to save the last change...